lib/sprockets/http_utils.rb

```diff
@@ -56,6 +56,7 @@ def parse_q_values(values)
     #
     # Returns Array of matched Strings from available Array or [].
     def find_q_matches(q_values, available, &matcher)
+      return [] if available == []
```

Comment:
We're allocating an array everytime this if code is called. We can speed it up by using a pre-allocated constant

```ruby
require 'benchmark/ips'

x = ""

EMPTY_ARRAY = [].freeze

Benchmark.ips do |x|
  x.report("constant") { x if x == EMPTY_ARRAY }
  x.report("literal ") { x if x == [] }
end
```

Gives us

```
Warming up --------------------------------------
            constant   326.041k i/100ms
            literal    306.879k i/100ms
Calculating -------------------------------------
            constant     10.675M (± 7.9%) i/s -     53.145M in   5.011048s
            literal       8.301M (± 7.7%) i/s -     41.429M in   5.022438s

Comparison:
            constant: 10675185.0 i/s
            literal :  8300648.6 i/s - 1.29x  slower
```